src/bundles/SellerRegistration/components/BusinessDetailsForm/BusinessDetailsForm.js

```diff
@@ -154,15 +168,77 @@ class BusinessDetailsForm extends BaseForm {
+                                                name={`address_line-${i}`}
+                                                id={`address_line-${i}`}
+                                                htmlFor={`address_line-${i}`}
+                                                label="Address"
+                                                messages={{
+                                                    required: 'You must provide address'
+                                                }}
+                                                validators={{required}}
+                                            />
+                                            <Textfield
+                                                model={`${model}.other_addresses[${i}].suburb`}
+                                                name={`suburb-${i}`}
+                                                id={`suburb-${i}`}
+                                                htmlFor={`suburb-${i}`}
+                                                label="Suburb"
+                                                messages={{
+                                                    required: 'You must provide a suburb'
+                                                }}
+                                                validators={{required}}
+                                            />
+                                            <Textfield
+                                                model={`${model}.other_addresses[${i}].state`}
+                                                name={`state-${i}`}
+                                                id={`state-${i}`}
+                                                htmlFor={`state-${i}`}
+                                                label="State"
+                                                messages={{
+                                                    required: 'You must provide a state'
+                                                }}
+                                                validators={{required}}
+                                            />
+                                            <Textfield
+                                                model={`${model}.other_addresses[${i}].postal_code`}
+                                                name={`postal_code-${i}`}
+                                                id={`postal_code-${i}`}
+                                                htmlFor={`postal_code-${i}`}
+                                                label="Postcode"
+                                                maxLength="4"
+                                                messages={{
+                                                    required: 'You must provide a postal code'
+                                                }}
+                                                validators={{required}}
```

Comment:
new limitNumbers validator for postcode is a more consistent experience than using pattern https://github.com/AusDTO/dto-digitalmarketplace-frontend/blob/5e3b21f26e7c8384e914b1b353414021ad8c1899/src/bundles/SellerRegistration/components/BusinessDetailsForm/BusinessDetailsForm.js#L154